gradle/libs.versions.toml

```diff
@@ -94,6 +94,12 @@ quickcheck = { module = "net.java:quickcheck", version.ref = "quickcheck" }
 spotless-plugin-gradle = { module = "com.diffplug.spotless:spotless-plugin-gradle", version.ref = "spotless" }
 turbine = { module = "app.cash.turbine:turbine", version = "1.0.0" }
 
+# Remove three-ten-abp once minSdkVersion is changed to 26 or later, and, instead use the
```

Comment:
To make it extra clear that this is NOT intended to be used as an actual dep, could you create a new section, something along the lines of `testonly` for additional visibility?

Reply:
Do you mean like this:

```
testonly-three-ten-abp = { ... }
```

Reply:
(i.e. add the "testonly-" prefix)

Reply:
exactly